terraform/modules/iam-users/developers.tf

```diff
@@ -18,18 +18,6 @@ resource "aws_iam_group_policy" "developers" {
       "Resource": [
         "arn:aws:iam::${var.aws_dev_account_id}:role/developers"
       ]
-    },
-    {
-      "Effect": "Allow",
-      "Action": [
-        "sts:AssumeRole"
-      ],
-      "Resource": "arn:aws:iam::*:role/packer",
-      "Condition": {
-        "Bool": {
-          "aws:MultiFactorAuthPresent": true
-        }
-      }
```

Comment:
A small pet peeve, we can leave trailing commas for the last item in a list in terraform, so that diffs are cleaner.

It is just a personal taste thing though so please feel free to ignore me.